features/step_definitions/git-flow.js

```diff
@@ -200,4 +186,22 @@ module.exports = function server() {
             Assert.oneOf(data.state, ['success', 'pending']);
         })
     );
+
+    // eslint-disable-next-line new-cap
+    this.After({
```

Comment:
Our tests don't care, but we need the `pipelineId`. The alternative (to creating a pipeline every time) would be searching through a list of pipelines for a particular `scmUri` after making a call to the scm to get a `repoId` to construct an `scmUri`.